client/utils/gatsby/layoutSelector.js

```diff
@@ -7,6 +7,19 @@ import {
 } from '../../src/components/layouts';
 import FourOhFourPage from '../../src/pages/404';
 
+const scriptAdd = (id, key, async, src) => {
+  var s = document.createElement('script');
```

Comment:
I think this need to change as `document` doesn't exist during SSR.

Reply:
@raisedadead I've double-checked this and I get
![DocumentSSR](https://user-images.githubusercontent.com/15801806/66958579-b6769300-f068-11e9-9ea5-1ccc38961090.png)
when I try to build. I don't understand how [this build](https://dev.azure.com/freeCodeCamp-org/freeCodeCamp/_build/results?buildId=220&view=results) succeeded.

Reply:
Thanks for catching this, I have reverted this PR.

Reply:
Thanks for catching this.
I will find another way.
Use helmet perhaps.